### DISCUSSION TITLE
Request for Custom key combination (hotkey) to active functions

### DISCUSSION THREAD

**Discussion by teitoku106** [kangyu-california/PersistentWindows#358] [Ideas]
![2024-October-07-1115-542-1126-PM](https://github.com/user-attachments/assets/ac383e6f-f08f-4bb2-a6d8-d420a3fb2c4d)

Hi, I have read through the PersistentWindows Quick Help ([URL](https://github.com/kangyu-california/PersistentWindows/blob/master/Help.md))

I'm requesting the ability to create custom key combination (hotkey) to active the app functions such as [Restores Windows from disk]

Example: 

- I can go to the app through GUI, set ctrl + shift + 1 as hotkey to active [Restores Windows from disk]

- Whenever I input the above key combination, it will active [Restores Windows from disk]

Is this possible? Because I have bind custom macro to my mouse for fast restore, it is easier for me to click the app icon in taskbar

**Comment by kangyu-california** [comment]
The requirement can be better served by using restore snapshot (which only takes one mouse click on the PW icon) instead of restore from disk.
Restore from disk is intended to be called only once after PC reboot, while restore snapshot is intended to be called throughout daily work, that's why only snapshot has command shortcut.

There are already too many hotkeys, one design goal of PW is to reduce the hotkey usage.
